tests/app/authentication/test_database_session_manager.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+
+from mock import patch, Mock, MagicMock
+from sqlalchemy.exc import IntegrityError
+
+from app.authentication.session_management import DatabaseSessionManager, EQ_SESSION_ID
+
+
+class TestDatabaseSessionManager(unittest.TestCase):
+
+    def test_store_user_id_rollback(self):
```

Comment:
Not sure what the preference is here for test methods? Do we make all test methods static or functions?

I guess grouping tests within a class is quite useful in some circumstances